src/frontend/components/pages/content/shoppingList/articles-by-group.pipe.ts

```diff
@@ -0,0 +1,20 @@
+import {Pipe, PipeTransform} from '@angular/core';
+import {Article} from '../../../../models/Article';
+
+@Pipe({name: 'articlesByGroup', pure: false})
```

Comment:
The reference to the array hasn't changed. It's the same array. That's all Angular cares about. From its perspective, same array, no change, no display update.